spec/support/vcr.rb

```diff
@@ -1,21 +1,21 @@
 VCR.configure do |c|
```

Comment:
Different endpoints maybe 10? But the params of the requests and the expected returned values are different pretty much each time.